src/main/java/com/google/sps/models/Stream.java

```diff
@@ -3,9 +3,12 @@
 
 public class Stream {
     ArrayList<TopChartsCluster> stream = new ArrayList<TopChartsCluster>();
+    ArrayList<StandardCluster> standardClusterStream = new ArrayList<StandardCluster>();
```

Comment:
I think we should just have a single List containing a bunch of clusters. Problem with this approach is the front end doesn't know the order of the clusters.